src/elitefurretai/utils/team_repo.py

```diff
@@ -13,14 +14,64 @@ class TeamRepo:
 
     def __init__(self, filepath: str = "data/teams"):
         self._teams: Dict[str, str] = {}
-        files = [
-            os.path.join(filepath, f)
-            for f in os.listdir(filepath)
-            if os.path.isfile(os.path.join(filepath, f))
-        ]
-        for file in files:
-            with open(file, "r") as f:
-                self._teams[os.path.basename(file).replace(".txt", "")] = f.read()
+
+        for format_folder in os.listdir(filepath):
+            format_name = format_folder
+            folder_path = os.path.join(filepath, format_folder)
+            if os.path.isdir(folder_path):
+                print(f"format name: {format_name}")
```

Comment:
Can you remove the prints? Or allow users to pass a `verbose` parameter if they want to print that defaults to off?

Reply:
Ah yes of course, I can't believe I didn't think to add a `verbose` parameter haha.